src/PluginUpdate.php

```diff
@@ -266,4 +267,26 @@ protected function feed_generation_migration(): void {
 
 		// Update done.
 	}
+
+	/**
+	 * Enable enhanced match for tags which have already been created.
+	 *
+	 * @return void
+	 */
+	protected function enable_enhanced_match(): void {
+		if ( ! $this->version_needs_update( '1.0.11' ) ) {
+			// Already up to date.
+			return;
+		}
+
+		$aem_enabled  = (bool) Pinterest_For_Woocommerce()::get_setting( 'enhanced_match_support', null );
+		$tracking_tag = Pinterest_For_Woocommerce()::get_setting( 'tracking_tag', null );
```

Comment:
Both of these don't require the second parameter.

```suggestion
		$aem_enabled  = (bool) Pinterest_For_Woocommerce()::get_setting( 'enhanced_match_support' );
		$tracking_tag = Pinterest_For_Woocommerce()::get_setting( 'tracking_tag' );
```